### DISCUSSION TITLE
Custom profile mode to a hotkey

### DISCUSSION THREAD

**Discussion by paslavskyit** [seerge/g-helper#1817] [Q&A]
Hello there, everyone. I see we can bind custom hotkey actions to run an application etc. but i don't quite understand how to bind my custom-made performance profile to switch on the hotkey. 
For example, i click on "Extra", and for Fn+C i want to switch exactly to "My_Mode_2". Any suggestions, please?

**Comment by seerge** [comment]
You can't do that. 

But you can use [default hotkeys ](https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings) to easily cycle between profiles forwards and backwards.

**Reply by MaxManus** [reply]
It would be a fantastic feature if implemented

**Reply by paslavskyit** [reply]
Instead of toggling profiles by M4 as i could have 7 or more and it takes time, so the hotkey would bind one exact profile that i need.

**Comment by paslavskyit** [comment]
> You can't do that.
> 
> But you can use [default hotkeys ](https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings) to easily cycle between profiles forwards and backwards.

yeah, sure. BTW, do you planning maybe to add this feature by any chance?

**Reply by seerge** [reply]
@shprd95 @MaxManus I have actually added 5 hotkeys for 3 built in + 2 (optional) custom modes

Please check build from this issue
https://github.com/seerge/g-helper/issues/1931
